db/data/regions/SJ.yml

```diff
@@ -5,6 +5,10 @@ name: Svalbard And Jan Mayen
 group: European Countries
 group_name: Europe
 code: SJ
+code_alternates:
+- "NO-SJ"
+- "NO-21"
+- "NO-22"
```

Comment:
```suggestion
```

`NO-21` is Svalbard without Jan Mayen.
`NO-22` is Jan Mayen without Svalbard.

Neither is a synonym for `NO-SJ`, any more than `US-NY` is a synonym for `US`.